<launch>
  <arg name="host" default="localhost"/>
  <arg name="port" default="2000"/>
  <arg name="fixed_delta_seconds" default="0.05"/>

  <group ns="carla">
    <node pkg="conformal_lattice_planner"
      type="ego_lane_following_node"
      name="ego_lane_following_planner"
      output="screen"
      required="true">

      <param name="host" value="$(arg host)"/>
      <param name="port" value="$(arg port)"/>
      <param name="fixed_delta_seconds" value="$(arg fixed_delta_seconds)"/>

      <remap from="~ego_plan" to="carla_simulator/ego_plan"/>
    </node>
  </group>
</launch>
